instrumentation/instrument.h

```diff
@@ -48,8 +49,10 @@ void bx_instr_before_execution(unsigned cpu, bxInstruction_c *i);
 #define BX_INSTR_EXIT(cpu_id) \
   bx_instr_exit(cpu_id)
 
-#define BX_INSTR_LIN_ACCESS(cpu_id, lin, phy, len, rw) \
-  bx_instr_lin_access(cpu_id, lin, phy, len, rw)
+#define BX_INSTR_LIN_ACCESS(cpu_id, lin, phy, len, memtype, rw) \
+                bx_instr_lin_access(cpu_id, lin, phy, len, memtype, rw)
```

Comment:
Could you make the indentation here consistent with the other #defines above and below (i.e. 2 spaces)?